css/css-ui/reference/transparent-accent-color-001-ref.html

```diff
@@ -4,38 +4,67 @@
 <link rel="help" href="https://drafts.csswg.org/css-ui-4/#widget-accent">
 <style>
 
-div {
-    border: solid orange;
-    padding: 1ch;
-    margin: 1ch;
-    float: left;
-}
-
-#t1 input { color-scheme: light;}
-#t1 input { accent-color: white; }
-
-#t2 input { color-scheme: light;}
-#t2 input { accent-color: #7f7fff; }
-
-#t3 { background: orange;}
-#t3 input { color-scheme: light;}
-#t3 input { accent-color: #7f7fff; }
-
+  div {
+      border: solid orange;
+      padding: 1ch;
+      margin: 1ch;
+      float: left;
+  }
+  
+  input { color-scheme: light;}
+  #extract-canvas{ accent-color: canvas; display:none;}
```

Comment:
I would just make it an empty `<div>` and `background-color` so you don't need to hide it.